.github/workflows/_reusable.rust.yml

```diff
@@ -37,6 +37,33 @@ jobs:
     container:
       image: xaynetci/yellow:v8
     timeout-minutes: 20
+    env:
+      XAYN_INGESTION__NET__BIND_TO: 127.0.1.1:3030
+      INGESTION_URI: http://127.0.1.1:3030
```

Comment:
Note that the `INGESTION_URI`, `PERSONALIZATION_URI` are just there for convenience.

You still need to start ingestion/personalization your self for the tests.

You can do so by running `cargo run --bin personalization` /`cargo run --bin ingestion`.

The default config should normally work and the `XAYN_INGESTION__NET__BIND_TO` and `XAYN_PERSONALIZATION__NET__BIND_TO` make sure it doesn't try to bind to the same port.

I will try to make it so that the default values for the port differ between services, but thats more complex as there is no context based config deserialization and they are both in the same libarary so using env variables of the binary during build time and similar do not work.